tests/test_cli_grids.py

```diff
@@ -0,0 +1,344 @@
+            "type": "line chart",
+            "summary": "This chart shows current average daily cases per"
+            " capita broken down by US state. Raw data from the New York"
+            " Times, calculations by Novem. Data last updated 23 November "
+            "2021",
+        },
+        {
+            "id": "covid_us_trend_region",
+            "shortname": "7N2Wv",
+            "created": "Thu, 17 Mar 2022 12:19:02 UTC",
+            "uri": "https://novem.no/g/7N2Wv",
+            "name": "Covid19 cases by US State",
+            "type": "area chart",
+            "summary": "This chart shows current average daily cases per"
+            " capita broken down by US state. Raw data from the New York"
+            " Times, calculations by Novem. Data last updated 23 November "
+            "2021",
+        },
+        {
+            "id": "en_letter_frequency",
+            "shortname": "QVgEN",
+            "created": "Thu, 17 Mar 2022 12:19:02 UTC",
+            "uri": "https://novem.no/g/QVgEN",
+            "name": "Letter frequency in the English language",
+            "type": "bar chart",
+            "summary": "Analysis of entries in the Concise Oxford dictionary"
+            " as published by the compilers. The chart above represents data"
+            " taken from Pavel Micka's website, which cites Robert Lewand's"
+            " Cryptological Mathematics.",
+        },
+        {
+            "id": "state_pop",
+            "shortname": "qNGgN",
+            "created": "Thu, 17 Mar 2022 12:19:02 UTC",
+            "uri": "https://novem.no/g/qNGgN",
+            "name": "Top 5 us states by population and age",
+            "type": "grouped bar chart",
+            "summary": "Historical unemployment rate in the Nordic countries."
+            " Data from IMFs World Economic Oulook published in October 2021"
+            " Chart last updated as of 25 January 2022",
+        },
+        {
+            "id": "unemployment_noridc",
+            "shortname": "2v1rV",
+            "created": "Thu, 17 Mar 2022 12:19:02 UTC",
+            "uri": "https://novem.no/g/2v1rV",
+            "name": "Historical Unemployment rates in the Nordic" " countries",
+            "type": "stacked bar chart",
+            "summary": "Historical unemployment rate in the Nordic "
+            "countries. Data from IMFs World Economic Oulook published in"
+            " October 2021 Chart last updated as of 25 January 2022",
+        },
+    ]
+
+    requests_mock.register_uri(
+        "get",
+        f"{api_root}vis/grids/",
+        text=json.dumps(grid_list),
+        status_code=200,
+    )
+
+    requests_mock.register_uri(
+        "get",
+        f"{api_root}u/demouser/g/",
+        text=json.dumps(user_grid_list),
+        status_code=200,
+    )
+
+    # try to list all grids
+    params = ["-g", "-l"]
+
+    # launch cli with params
+    sys.argv = ["novem"] + params
+
+    # run cli
+    run_cli()
+    out, err = capsys.readouterr()
```

Comment:
I put a nice little `cli` fixture in `conftest.py`, see usage in https://github.com/novem-code/novem-python/blob/main/tests/test_cli_config.py#L61